File: MB Website Final Bun/classes/manager.php
<?php
require_once('user.php');
class Manager extends User
{
    //variables
    protected $position;

    //constructor
    public function __construct($id, $name, $gender, $password, $email, $address, $phoneNumber, $hourlyWage, $hoursWorked, $birthDate, $holidayDays, $sickDays)
    {
        $this->position = $position;
        parent::__construct($id, $name, $gender, $password, $email, $address, $phoneNumber, $hourlyWage, $hoursWorked, $birthDate, $holidayDays, $sickDays);
    }

    //getters
    public function GetPosition()
    {
        return "$this->position";
    }

    //setters

    public function SetPosition($position)
    {
        $this->position = $position;
    }

}

File: MediaBazaar_Code_Final/MediaBazaar/classes/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MediaBazaar.classes
{
    public class Manager : User
    {
        //instance variables
        protected String position;

        //constructor 
        public Manager(int id, string name, String password, string email, String gender, string phoneNumber, string address, double hourlyWage, double hoursWorked, DateTime birthDate, int holidayDays, int sickDays, String position) :
            base(id, name, password, email, gender, phoneNumber, address, hourlyWage, hoursWorked, birthDate,  holidayDays, sickDays)
        {
            this.position = position;
        }

        //properties
        public String Position
        {
            get { return this.position; }
            set { this.position = value; }
        }

        //methods
        public override String ToString()
        {
            return $"Manager: ID - {id}, Name - {name}, Email - {email}, Phonenumber - {phoneNumber}";
        }
    }
}
